cmd/corald/handlers/action.go

```diff
@@ -0,0 +1,151 @@
+package handlers
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/ardanlabs/kit/web"
+	"github.com/coralproject/shelf/cmd/corald/service"
+)
+
+// actionHandle maintains the set of handlers for the action api.
+type actionHandle struct{}
+
+// Action fronts the access to the action service functionality.
+var Action actionHandle
+
+var (
+	// ErrParameterNotFound is action not found error.
+	ErrParameterNotFound = errors.New("Parameter not found")
+
+	// ErrActionNotFound is action not found error.
+	ErrActionNotFound = errors.New("Action not found")
+
+	//ErrActionNotAllowed is an error when the action is not allowed.
+	ErrActionNotAllowed = errors.New("Action not allowed")
+
+	//ErrTypeNotExpected comes when the type asserted was not expected
+	ErrTypeNotExpected = errors.New("Type not expected")
+)
+
+//==============================================================================
+
+// Create an action (flag, like, etc) on an Item.
+// 200 Success, 404 Not Found, 500 Internal
+func (actionHandle) Add(c *web.Context) error {
```

Comment:
Comment on this function needs to start with "Add".  Linting should capture this if you run `golint`